### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by brhanuhailu**
## This is pull request I applied:
- [X] Set up a new project using webpack for to do list.
- [X] Used  /src directory for source files (index.html, index.js and style.css)
- [X]  Used /dist directory for distribution files generated by webpack 
- [X] Created an array of tasks with index, completed and description.
- [X] Displayed array of tasks dynamically.
- [X] Created style.css, set rules and loader it with Webpack Style/CSS Loader.
- [X] Created list dynamically
- [X] Linter error checked

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,10 +2,24 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;1,100&display=swap" rel="stylesheet">
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <div class="all-container">
+    <div class="container">
+        <div class="card-layout">
+            <div class="card-title"> <p>Today`s To Do</p> <i class="fa-solid fa-rotate"></i> </div>
+            <div class="sub-title"><em> Add to your list... </em></div>
```
Comment (osamaashraf6) [inline src/index.html]: - [ ] You did well 😃 at this milestone, but I see here that you make the text add todo static and we want it to be input to let the user can write the todo, so kindly 😊 try to add the input here to be like the original design to let the users can write their todos

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| Recording 2023-02-22 at 13 53 26   |     Recording 2023-02-22 at 13 54 37   |

File: README.md [hunk 2]
```diff
@@ -1 +1,186 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT]https://github.com/brhanuhailu/to-Do-list.git) licensed.
+(Check the LICENSE file)
```
Comment (osamaashraf6) [inline README.md]: - [ ] you did a great job 😃 at adding the MIT LICENCE, but I see that when clicking on the link to open the page of LICENCE I see that it is redirecting me to the page of your todo page in GitHub, not to the licence page, so please 😊 try to put the correct link for the MIT LICENCE to let anyone when clicking on the link of LICENCE redirect him/her to the page of the LICENCE 💯

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @brhanuhailu 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️

- [x] Nice design ✔️
- [x] Descriptive READme file in a professional way.✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️





## Required Changes ♻️

- check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html (hunk 1 above)
Comment (brhanuhailu) [reply src/index.html:18]: Dear @osamaashraf6 thank you for your constructive comments. but I am little bit confusing with the tamplate, this the first setup the next day add and remove project will be done next to this project. may I get some inforamtion please?

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @brhanuhailu 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**